src/App.js

```diff
@@ -50,12 +49,11 @@ class App extends React.Component {
       homeComponent = () => <Redirect to={{ pathname: '/onboarding' }} />;
     } else if (!signedIn) {
       homeComponent = Login;
-    } else if (isGeneral && isSubscriber) {
-      homeComponent = SubscriberWithSharesDashboard;
-    } else if (isGeneral) {
-      homeComponent = GeneralDashboard;
     } else if (isSubscriber) {
+      // Subscriber with shares is accounted for in dashboard
```

Comment:
maybe update this comment to be more applicable to devs with no context:

```suggestion
      // Dashboard for both subscriber and subscriber ownrers (subscribers with shares)
```